source/extensions/filters/network/ext_proc/ext_proc.h

```diff
@@ -34,17 +34,17 @@ class Config {
     return processing_mode_;
   }
 
-  const absl::optional<const envoy::config::core::v3::GrpcService> grpcService() const {
+  const OptRef<const envoy::config::core::v3::GrpcService> grpcService() const {
```

Comment:
@botengyao It wasn't clear to me why this returns an optional instead of just a reference, but I guess there's a good reason.
I only changed it from optional to OptRef, but let me know if this should be changed to a simple reference.

Reply:
The original thinking is gRPC service is just one of the interfaces, but I think we can make it ref and remove the default one right now.

Reply:
Just to make sure I understand. Do you mean that the [NetworkExtProcFilter code](https://github.com/envoyproxy/envoy/blob/8055d56c5123ebbe4fad75572ad32c00bb69706c/source/extensions/filters/network/ext_proc/ext_proc.cc#L11-L12 should also be changed and remove the default gRPC-service?

Reply:
sry, I think a better logic is to reject a config without a gRPC service, and we don't need the default one, since it will not work anyway, and then we can use a ref without optional. But for this PR it should be good.